engines/bastion_katello/app/assets/javascripts/bastion_katello/products/new/views/product-new-form.html

```diff
@@ -33,7 +33,10 @@
             ng-options="syncPlan.id as syncPlan.name for syncPlan in syncPlans">
       <option value=""></option>
     </select>
-    <a ui-sref="products.new.sync-plan" translate>+ New Sync Plan</a>
+    <a ui-sref="products.new.sync-plan" translate>
+      <i class="pficon pficon-add-circle-o"></i>
```

Comment:
@waldenraines Thanks for your comments. 
Based on the discussion on the foreman-dev group. 
We suggested to keep consistent to using the same icon for the same button. For this time change, if the button already include an icon, then i updated the icon to PatternFly Create Icon, if the button doesn't have an icon, then I just update the label from New to Create.

So here, "+ New Sync Plan", it has an icon "+",  so I updated the "+" icon to the PatternFly Create icon.

Currently using:
![screenshot from 2017-01-11 13-19-46](https://cloud.githubusercontent.com/assets/701009/21836617/834ac5dc-d801-11e6-9b86-4da5fae70ff4.png)

After update:
![screenshot from 2017-01-11 13-28-06](https://cloud.githubusercontent.com/assets/701009/21836649/d49ec4ce-d801-11e6-9356-5017a98ee928.png)